src/core/thread/mesh_forwarder.cpp

```diff
@@ -1045,6 +1045,18 @@ void MeshForwarder::HandleReceivedFrame(Mac::RxFrame &aFrame)
             LogFrame("Received empty payload frame", aFrame, OT_ERROR_NONE);
         }
 
+#if OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2 && OPENTHREAD_FTD
+        // From Thread 1.2, MAC Data Frame can also act as keep-alive message if child supports
```

Comment:
Wasn't this functionality added with #4325 

https://github.com/openthread/openthread/blob/fe6a7e1493ef8dcbccbe19b5bcb94e377a41a848/src/core/mac/mac.cpp#L1678-L1685

Reply:
rebase caused issue..